Firmware/Marlin_main.cpp

```diff
@@ -6389,14 +6389,24 @@ SERIAL_PROTOCOLPGM("\n\n");
         if(code_seen('R')) print_time_remaining_normal = code_value();
         if(code_seen('Q')) print_percent_done_silent = code_value();
         if(code_seen('S')) print_time_remaining_silent = code_value();
+#if 1
+        if(code_seen('C')){
+            float print_time_to_change_normal_f = code_value_float();
```

Comment:
Also
https://github.com/prusa3d/Prusa-Firmware/blob/MK3_3.10.0/Firmware/Marlin_main.cpp#L319-L320
and
https://github.com/prusa3d/Prusa-Firmware/blob/MK3_3.10.0/Firmware/Marlin.h#L369-L370
should be float

Reply:
no, that's what I want to avoid